torchvision/ops/focal_loss.py

```diff
@@ -32,6 +32,13 @@ def sigmoid_focal_loss(
         Loss tensor with the reduction option applied.
     """
     # Original implementation from https://github.com/facebookresearch/fvcore/blob/master/fvcore/nn/focal_loss.py
+
+    # Check reduction option
+    if reduction != "none" or reduction != "mean" or reduction != "sum":
+        raise ValueError(
+            f"Invalid Value for arg 'reduction': '{reduction} \n Supported reduction modes: 'none', 'mean', 'sum'"
+        )
```

Comment:
Could you please move this on lines 53-56 similar to the other loss functions?

Reply:
Will do

Reply:
I have also made sure that all the loss functions follow the same idiom at checking the reduction value.